src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -134,4 +134,61 @@ let Test (setTestParams: TestParams -> TestParams) projects =
             then
                 failwithf "Test failed on %s" args
     finally
-        traceEndTask "DotNet.Test" ""
\ No newline at end of file
+        traceEndTask "DotNet.Test" ""
+
+
+/// DotNet pack parameters
+type PackParams = {
+    /// ToolPath - usually just "dotnet"
+    ToolPath: string
+
+    /// Working directory (optional).
+    WorkingDir: string
+
+    /// A timeout for the command.
+    TimeOut: TimeSpan
+    
+    /// The build configuration.
+    Configuration : string
+}
+
+let private DefaultPackParams : PackParams = {
+    ToolPath = commandName
+    WorkingDir = Environment.CurrentDirectory
+    Configuration = "Release"
+    TimeOut = TimeSpan.FromMinutes 30.
+}
+
+/// Runs the dotnet "pack" command.
+/// ## Parameters
+///
+///  - `setPackParams` - Function used to overwrite the pack default parameters.
+///
+/// ## Sample
+///
+///     !! "src/test/project.json"
```

Comment:
why `src/test/project.json`? pack work on the directory, like all dotnet commands, not on the file